docs/content/commands.md

```diff
@@ -70,34 +72,37 @@ public class BuildSystem : ComponentSystem
     {
         for(var i = 0; i < data.Length; i++)
         {
-            var requestSender = data.BuilderRequestSender[i];
+            var requestSender = data.BuildWallSender[i];
             var entityId = data.EntityIds[i];
             
-            BuildRequest buildRequest = new BuildRequest
+            Builder.BuildWall.Request request = new Builder.BuildWall.Request
             {
-                Location = new Location(...),
-                Rotation = new Rotation(...)
-            }
+                TargetEntityId = entityId,
+                Payload = new BuildRequest
+                {
+                    Location = new Location(...),
+                    Rotation = new Rotation(...)
+                }
+            };
+
             
-            requestSender.SendBuildWallRequest(entityId, buildRequest);
+            requestSender.RequestsToSend.Add(request);
         }
     }
 }
 ```
 
 This system runs on a client. It injects all SpatialOS entities in the client's view that have the `Builder` SpatialOS component, and sends a `build_wall` command to each SpatialOS entity (which will be received by the managed worker).
```

Comment:
I know you didn't edit this but...
(1) It injects what in the SpatialOS entities? 
Or does it inject the SpatialOS entites *into* the client' view?
I can't follow the sentence.
(2) Does it send the `build_wall` command to each SpatialOS entity which has the `Builder` component or all of them. Currently it says "each" which suggests all of them. I suspect it's the one with the components only?
(3) Which managed worker (server-worker) receives the commands?  Or is it just general "all server workers"?
(4) We need to use the current terminology:
"client-worker", "managed-worker".

So something like the following (but my interpretation may be out).

"This system runs on a client-worker. It injects <something into??> all SpatialOS entities currently in the client-worker's view that have the `Builder` SpatialOS component. It then sends a `build_wall` command to each SpatialOS entity with that component. All server-workers receive this command."

Reply:
1. It doesn't inject anything into the entities, it (Unity's ECS impl) injects the entities into the system . 
2. It sends a `build_wall` command to every entity that the worker can see that has a `Builder` component
3. It actually depends - I'm going to remove this as its dependent on many other factors 
4. Going to drop it. 

Also - the system doesn't specify which worker this runs on, in fact, as its written now it will run on any worker which has an entity checked out with the `Builder` component. Will edit to make it more clear.

Will rephrase as:

"This system is an example of sending command requests. Unity injects all ECS entities which have [list of components] into this system before `OnUpdate`. This system iterates over the injected entities and sends a `build_wall` command request to each of them."